ethereum/ethereum_type.bal

```diff
@@ -0,0 +1,303 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+# EthereumConfiguration is used to set up the Ethereum configuration. In order to use this connector, you need
+# to provide the valid JSON-RPC endpoint , version and network id.
+# + jsonRpcVersion - JSON RPC version
```

Comment:
Require a new line between description and fields. Please do the same for other similar occurrences.

Reply:
Fixed